rest/data/project.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/mongodb/grip"
 	"github.com/mongodb/grip/message"
 	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2"
```

Comment:
nope! can't have dependencies to the driver for things like this.

Reply:
What about the regular mongo driver? Otherwise I can't figure out how to make the mock error match, so we either can't test this case, or we can check strings.Contains(err.Error(), "not found" instead of adb.ResultsNotFound.